docs/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF

```

Comment:
Do we need that bat?

Reply:
Sphinx makes it automatically in case we wanted to build docs on Windows, but since we don't I can remove it.